src/com/facebook/buck/apple/project_generator/ProjectGenerator.java

```diff
@@ -1503,6 +1506,27 @@ private PBXNativeTarget generateBinaryTarget(
     return target;
   }
 
+  private ImmutableList<String> getExcludesForModulesUnderTests(
+      TargetNode<? extends CxxLibraryDescription.CommonArg, ?> targetNode) {
+    ImmutableList.Builder<String> testingOverlayBuilder = new ImmutableList.Builder<>();
+    visitRecursivePrivateHeaderSymlinkTreesForTests(
+        targetNode,
+        (nativeNode, headerVisibility) -> {
+          // If we are testing a modular apple_library, we expose it non-modular. This allows the
```

Comment:
If I understand correctly, this implies that testing targets written in Obj-C cannot `@import` the libs under testing and have to resort to `#import`. Is that right?

Reply:
Yes, that is correct. It seems non-ideal but after some investigation into how Xcode handles this this seems like the way to go. I couldn't find any configuration of settings in a hand made xcode project which would allow you to use both import styles.